docs/01_introduction.md

```diff
@@ -55,23 +53,21 @@ consistent testing of incoming contributions, and thorough regression testing be
 EasyBuild is ***not*** **YABT (Yet Another Build Tool)**: it does not replace established build
 tools like CMake or ``make``, it wraps around them.
 If the installation procedure of a software package involves running some unholy trinity of tools while whispering
-the correct magic incantations, EasyBuild just automates this process so you don't have to go through it manually.
+the correct magic incantations, EasyBuild seeks to automate this process for you.
 
 It is ***not*** **a replacement for traditional Linux package manangers** like ``yum``, ``dnf`` or ``apt``.
-EasyBuild leverages some tools and libraries provided by OS, including glibc, OpenSSL, Infiniband & GPU drivers, etc.,
-so you will still need to use the package management tool that comes with your Linux distribution of choice
-for installing (system) software.
+EasyBuild relies on certain tools and libraries provided by the operating system. This includes glibc, OpenSSL, Infiniband, GPU drivers, and so on. It is required that these tools are installed and managed by other means. This is typically done via the package management tool that comes with your Linux distribution.
 
 Finally, EasyBuild is ***not a magic solution to all your (software installation) problems**.
-* a magic solution to all your (software installation) problems: you may (and will) still run into compiler errors (unless somebody has already taken care of it);
+You may, and probably will still, run into compiler errors unless somebody has already taken care of the problem for you.
 
 
 ### *Implementation*
 
 <img src="https://boegel.github.io/easybuild-tutorial/img/Python-logo.png" style="border-right: 20px solid white;border-top: 5px solid white; float:left" width="80px"/>
 
 
-EasyBuild is **implemented in [Python](https://www.python.org/)**, and supports both Python 2.7 and 3.5+.
+EasyBuild is **implemented in [Python](https://www.python.org/)**, and supports both Python 2.7 and 3.5+. Python 2.7 support is required for older environments where 3.5+ is unavailable and the the testing suite for EasyBuild retains the same coverage Python 2.7 as for 3.5+.
```

Comment:
That last sentence isn't fully true, it's mostly about legacy and not hard breaking backwards compatibility and forcing everyone to use Python 3.

Even on RHEL8, Python 2.7 is still available, so strictly speaking we didn't actually need to support Python 3 yet. ;)

Why did you feel the need to extend this part a bit?

Reply:
I wanted to state that the 2.7 was a fully supported version, not just "supported". I took the original and added fully. Much better.